### PR TITLE
Presences

### LINKED ISSUE #18: Presences
Presences for Subjects

### PR DESCRIPTION
**Pull request by astutus**
Closes #18

### CODE REVIEW

File: src/main/java/pl/edu/agh/ki/mwo/SchoolWebApp/entity/Presence.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import org.hibernate.annotations.NotFound;
+import org.hibernate.annotations.NotFoundAction;
+
+@Entity
+@Table(name = "presences")
+public class Presence implements java.io.Serializable {
+
+	private Subjects subject;
+	private Student student;
+	/*@Column(name = "date")
```
Comment (Yankowski) [inline src/main/java/pl/edu/agh/ki/mwo/SchoolWebApp/entity/Presence.java]: remove commented line

File: src/main/java/pl/edu/agh/ki/mwo/SchoolWebApp/entity/Presence.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+	}
+
+	public Calendar getDateField() {
+		return dateField;
+	}
+
+	public void setDateField(Calendar dateField) {
+		this.dateField = dateField;
+	}
+
```
Comment (Yankowski) [inline src/main/java/pl/edu/agh/ki/mwo/SchoolWebApp/entity/Presence.java]: remove uneccesary lines

File: src/main/java/pl/edu/agh/ki/mwo/SchoolWebApp/entity/Presence.java [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 import java.util.Calendar;
 
+import javax.persistence.CascadeType;
 import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.FetchType;
```
Comment (Yankowski) [inline src/main/java/pl/edu/agh/ki/mwo/SchoolWebApp/entity/Presence.java]: remove unused imports

File: src/main/java/pl/edu/agh/ki/mwo/SchoolWebApp/controller/PresenceController.java [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+    private PresenceRepository presenceRepository;
+
+    @RequestMapping(value = "/Presences")
+    public String subjectList(Model model) {
+
+        model.addAttribute("presencesList", presenceRepository.findAll());
+        return "presencesList";
+    }
+
+/*    @RequestMapping(value = "/AddSubject")
```
Comment (Yankowski) [inline src/main/java/pl/edu/agh/ki/mwo/SchoolWebApp/controller/PresenceController.java]: remove commented code

File: src/main/java/pl/edu/agh/ki/mwo/SchoolWebApp/controller/PresenceController.java [hunk 5]
```diff
@@ -44,16 +44,16 @@ public String displayAddSchoolClassForm(Model model, HttpSession session) {
...
-        subjectRepository.deleteById(Long.valueOf(subjectId));
-        model.addAttribute("subjectsList", subjectRepository.findAll());
+        presenceRepository.deleteById(Long.valueOf(presenceId));
+        model.addAttribute("presencesList", presenceRepository.findAll());
 
-        return "subjectList";
+        return "presencesList";
     }
-
+/*
```
Comment (Yankowski) [inline src/main/java/pl/edu/agh/ki/mwo/SchoolWebApp/controller/PresenceController.java]: remove commented code

File: src/main/java/pl/edu/agh/ki/mwo/SchoolWebApp/controller/PresenceController.java [hunk 6]
```diff
@@ -1,5 +1,9 @@
 package pl.edu.agh.ki.mwo.SchoolWebApp.controller;
```
Comment (Yankowski) [inline src/main/java/pl/edu/agh/ki/mwo/SchoolWebApp/controller/PresenceController.java]: remove unused imports

**Review by Yankowski — CHANGES_REQUESTED** [review]
Everythins looks ok, the calendar looks great, please remove unused improts and it is ready to merge, do not pay attention on previous commits becasue as far as i see this was resolved in the latest commits.

**Comment by bockowsk** [comment]
ok, a new commit with some cleaning

**Pull request merged**